hash/adam_cart.xml

```diff
@@ -22,7 +22,7 @@ license:CC0
 	-->
 
 	<software name="ard" supported="no">
-		<description>ADAM Resident Debugger (Prototype Rev20)</description>
+		<description>ADAM Resident Debugger (prototype rev20)</description>
```

Comment:
I checked the ROM for strings for one of these, and it seems to say “REVISION 2.0” – rev20 was probably an abbreviation for some more limited field.  All the text is uppercase in the ROM, so it’s probably not a stylistic choice to use uppercase.